model/config.go

```diff
@@ -3098,7 +3098,7 @@ type Config struct {
 	GuestAccountsSettings     GuestAccountsSettings
 	ImageProxySettings        ImageProxySettings
 	CloudSettings             CloudSettings  // telemetry: none
-	FeatureFlags              *FeatureFlags  `json:",omitempty"` // telemetry: none
+	FeatureFlags              *FeatureFlags  `access:"any" json:",omitempty"` // telemetry: none
```

Comment:
Any tag is fine but we definitely need documentation somewhere explaining what it is and how it works. At first my first intuition was that ANYBODY will get it no matter what but then I realized it's any system admin.

Reply:
All access tags only apply to System admin

Reply:
All access tags apply to all system roles.

Reply:
All access tags apply to all roles because they're based on the permissions.

Doesn't each field in `FeatureFlags` also need an access tag?